MAUI/DataGrid/loadMore.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: Load More in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Load More support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Load More in MAUI DataGrid (SfDataGrid)
+
+The DataGrid enables the LoadMore option when the `SfDataGrid.AllowLoadMore` property is set to `true`, and also by setting the SfDataGrid.`LoadMoreCommand` property. When LoadMore is enabled, the control loads a subset of data to its data source at runtime using DataGridLoadMoreView.
+
+When the grid reaches maximum offset while scrolling down, an interactive DataGridLoadMoreView is displayed. Tapping the load more view triggers a command to add more data to the data source of the grid at runtime.
+
+## Load more command
+The DataGrid loads records to its data source at runtime by triggering an ICommand bound to the `SfDataGrid.LoadMoreCommand` property. When the DataGridLoadMoreView is tapped, the CanExecute of the ICommand returns true, and this command is triggered to load the records at runtime.
+
+Set the `SfDataGrid.IsBusy` property to true before loading items to notify the grid that more items are to be loaded. Set the property to false after loading items into the grid. When loading items, alter the time for the LoadMore animation from the sample by setting a delay based on the requirement.
```

Comment:
The DataGrid loads a subset of data to its data source at runtime by triggering an ICommand bound to the `SfDataGrid.LoadMoreCommand` property.  It will be executed when the user taps the load more view.

Set the `SfDataGrid.IsBusy` property to true before loading items to notify the grid that more items are about to be loaded. Set the property to false after successfully loading items into the grid. When loading items, customize the duration for displaying the activity indicator by introducing a delay based on specific requirements.

Reply:
Replaced the content